miden-lib/asm/sat/account.masm

```diff
@@ -218,3 +221,43 @@ export.set_code
     exec.layout::set_new_acct_code_root
     # => []
 end
+
+#! Gets an item from the account storage. Panics if the index is out of bounds.
+#!
+#! Stack: [index]
+#! Output: [VALUE]
+#!
+#! - index is the index of the item to get.
+#! - VALUE is the value of the item.
+export.get_item
+    # get the storage root
+    exec.layout::get_acct_storage_root
+    # => [storage_root, index]
+
+    # get the item from storage
+    movup.4 push.STORAGE_TREE_DEPTH mtree_get
+    # => [VALUE, ROOT]
+end
+
+#! Sets an item in the account storage. Panics if the index is out of bounds.
+#!
+#! Stack: [index, V']
+#! Output: [R', V]
```

Comment:
Why are we returning the new root from here?

Reply:
It provides the user access to the new root "for free" in case they need to use it.  It would probably be a cleaner interface if we drop it. Happy to do so, what do you think?

Reply:
Updated solution drops the new root.